lib/ansible/modules/net_tools/dnsmadeeasy.py

```diff
@@ -94,8 +94,6 @@
   systemDescription:
     description:
       - Description used by the monitor.
```

Comment:
You should mention in the description that it is required if `monitor` is `true`.